src/app/TargetSelect/TargetSelect.tsx

```diff
@@ -249,7 +254,7 @@ export const TargetSelect: React.FunctionComponent<TargetSelectProps> = (props)
             aria-label="Select Input"
           >
           {
-            ([<SelectOption key='placeholder' value='Select Target...' isPlaceholder={true} />])
+            ([<SelectOption key='placeholder' value='Select Target...' isPlaceholder={true} itemCount={numTargets} />])
```

Comment:
Do we need `numTargets` as a separate piece of state here? Or can we just pass `itemCount={targets.length}` perhaps?

Reply:
Yeah you're right, I'm overcomplicating...